docker/server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# NOTE: The docker build context directory must be the root of the source tree.
+# NOTE: To build the release image, SQLFlow must be built into ./build
+FROM ubuntu:18.04
+
+# Install and configure the pre-built SQLFlow binaries.
+COPY build /build
+ENV SQLFLOW_PARSER_SERVER_PORT=12300
+ENV SQLFLOW_PARSER_SERVER_LOADING_PATH="/usr/local/sqlflow/java"
+
+# This image includes the SQLFlow gRPC server and external parser servers.  The
+# latter is in Java, so we need JRE.
+RUN apt-get -qq update \
+	&& apt-get -qq install -y openjdk-8-jre-headless \
+	&& mkdir -p "$SQLFLOW_PARSER_SERVER_LOADING_PATH" \
+	&& mv /build/*.jar "$SQLFLOW_PARSER_SERVER_LOADING_PATH" \
```

Comment:
We should  install the wheel package in server image:
``` bash
pip install --quiet /build/*.whl \
```

Reply:
I noticed that there are three wheel files in `./build`:

- couler*.whl
- fluid*.whl
- sqlflow_models*.whl

I think only the step image needs them, am I right?

Reply:
Server relies on couler to generate YAML file. So, we should install couler. We also need  to 
 install cubectl.

Reply:
Oh. Yes! Thanks for the reminder!